src/container_support/serving.py

```diff
@@ -134,6 +139,18 @@ def _download_user_module_internal(cls, env):
                 pass
             raise
 
+    @staticmethod
+    def _create_nginx_config(serving_env):
+        template = cs.utils.read_file(nginx_config_template_file)
+        pattern = re.compile(r'%(\w+)%')
+
+        template_values = {
+            'NGINX_HTTP_PORT': serving_env.port
+        }
+        config = pattern.sub(lambda x: template_values[x.group(1)], template)
+        logger.info('nginx config: \n%s\n', config)
+        cs.utils.write_file(nginx_config_file, config)
```

Comment:
this path is inside package resources... we should not be modifying things inside our running package.
put the new conf file. use /etc/nginx/nginx.conf (overwriting) or /etc/nginx.conf instead.

Reply:
Ahhh... gotcha, don't modify the package itself. Makes a lot of sense. I'll use /etc/nginx.conf, should we have some sort of sagemaker identifier in the directory? like /etc/sagemaker/nginx.conf?

Reply:
Spoke offline, using /etc/sagemaker-nginx.conf